CSharp/BatchExplorer/Converters/EnumToStringConverter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Data;
+
+namespace Microsoft.Azure.BatchExplorer.Converters
+{
+    public class EnumToStringConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            return value.ToString();
```

Comment:
null check here?  Also, if you really want it to be an "EnumToSTringConverter" as opposed to an "ObjectToStringConverter" you could check that it was of type Enum.